airbyte-proxy/test.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+NAME="airbyte-proxy-test-container"
+PORT=18000
+BASIC_AUTH_USERNAME=airbyte
+BASIC_AUTH_PASSWORD=password
+BASIC_AUTH_UPDATED_PASSWORD=pa55w0rd
+
+function start_container () {
+  docker run -d -p $PORT:8000 --env BASIC_AUTH_USERNAME=$1 --env BASIC_AUTH_PASSWORD=$2 --name $NAME airbyte/proxy:dev
+  docker run --rm jwilder/dockerize -wait tcp://host.docker.internal:$PORT -timeout 30s
+  sleep 2
+  # while ! `nc -z -v -w30 localhost $PORT`; do
+  #   echo -n "."
+  #   sleep 1;
+  # done
+}
+
+function stop_container () {
+  echo "Stopping $NAME"
+  docker stop $NAME
+  docker rm $NAME
+}
+
+# a local version of airbyte/proxy:dev should already have been
+echo "Testing airbyte proxy..."
+
+stop_container;
```

Comment:
is this in case a current Airbyte Proxy container is running?

Reply:
Yep!